wscript_build.py

```diff
@@ -431,7 +432,7 @@ def build(ctx):
         ( "video/out/vo_lavc.c",                 "encoding" ),
         ( "video/out/vo_rpi.c",                  "rpi" ),
         ( "video/out/vo_null.c" ),
-        ( "video/out/vo_opengl.c",               "gl" ),
+        ( "video/out/vo_gpu.c",                  "gpu" ),
         ( "video/out/vo_opengl_cb.c",            "gl" ),
         ( "video/out/vo_sdl.c",                  "sdl2" ),
         ( "video/out/vo_tct.c" ),
```

Comment:
This hole commit sure could have been reduced in size by making all those annoying changes separate or delying them (for example why the int->bool return value change?).

Reply:
`int` -> `bool`: modernize codebase. I consider functions that return “int” where “bool” would be fine to be MPlayer legacy. The opengl/gpu rename was done right away so I could fix the headers in one go without having to go through the bother of re-testing this crap on platforms I don't even own

Reply:
Most of them returned an error code (of which only -1 was defined).

Reply:
I would personally never return an `int` blindly, I would always give it a defined name, like `enum foo_err`. The reason is simple: if I see a function with an `int` return, I want a name I can jump to (`ctags`) in order to find the enumeration of all possible values.

Anecdote: For the longest time ever I never even know the `control()` functions had an `enum { VO_OK = 0, ... }` in vo.h, because all I ever saw was the `control()` implementation and not the corresponding header, and saw them returning random shit like `-2` which I didn't know what the fuck it was supposed to mean because there was no name to ctags jump to and the legacy MPlayer crap didn't use the VO_ERROR etc. names.